OpenStreetMapViewer/src/main/java/org/osmdroid/samplefragments/geopackage/GeopackageFeatureTiles.java

```diff
@@ -245,9 +245,9 @@ private void updateInfo() {
      *
      * @return
      */
-    protected static Set<File> findMapFiles() {
+    protected Set<File> findMapFiles() {
```

Comment:
I know we're talking only about samples, but that's not that great to see the same code in `GeopackageFeatureTiles`, `GeopackageFeatures`, `GeopackageSample` and `MapsforgeTileProviderSample`.

What about implementing `public static Set<File> findMapFiles(Activity activity, String endsWith)`? Somewhere in the sample app (perhaps `BaseFragment` - and we would even get rid of `static` and the `Activity` parameter).

Reply:
I actually started a cleanup off the sample app (formatting, lint errors etc.). After that I wanted to take a look at things like this. 👍 
But that's for another PR I think...